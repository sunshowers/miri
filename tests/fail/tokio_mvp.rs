//@compile-flags: -Zmiri-disable-isolation
//@error-pattern: can't call foreign function: epoll_create1
//@normalize-stderr-test: "   = note: inside .*\n" -> ""

#[tokio::main]
async fn main() {}
