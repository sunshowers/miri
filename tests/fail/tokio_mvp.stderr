error: unsupported operation: can't call foreign function: epoll_create1
  --> /home/ubuntu/.cargo/registry/src/github.com-1ecc6299db9ec823/mio-0.8.4/src/sys/PLATFORM/selector/epoll.rs:LL:CC
   |
LL |         syscall!(epoll_create1(flag)).map(|ep| Selector {
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't call foreign function: epoll_create1
   |
   = help: this is likely not a bug in the program; it indicates that the program performed an operation that the interpreter does not support
   = note: backtrace:
note: inside `main` at $DIR/tokio_mvp.rs:LL:CC
  --> $DIR/tokio_mvp.rs:LL:CC
   |
LL | #[tokio::main]
   | ^^^^^^^^^^^^^^
   = note: this error originates in the macro `syscall` (in Nightly builds, run with -Z macro-backtrace for more info)

note: some details are omitted, run with `MIRIFLAGS=-Zmiri-backtrace=full` for a verbose backtrace

error: aborting due to previous error

